### ISSUE TITLE
1.16.5?

### ISSUE THREAD

**Issue by hxr404** [THEREALWWEFAN231/WWEs-Altening-Mod#7]
Thanks for creating it, but is there a 1.16 (fabric) Version?
if there isn't one could you make it pls, I really need it, as I'm on Linux

**Comment by hcesaropz** [comment]
yes, i know this post is (sort of) necrobumping, but i tried to update this mod to 1.17.1

![image](https://user-images.githubusercontent.com/61612754/190359281-840829f1-6985-44af-a75d-eb1f79e269b4.png)

sadly, Gradle refuses to compile properly

**Comment by dolphin-cat** [comment]
@DarwinUang could you please send me the file or upload your fork to github so that i can clone it?

**Comment by hcesaropz** [comment]
> @DarwinUang could you please send me the file or upload your fork to github so that i can clone it?

[DarwinUang/WWEs-Altening-Mod](https://github.com/DarwinUang/WWEs-Altening-Mod)

i actually lost the original fork i made after "accidentally" deleting my entire data partition
so here's a new fork, again with commits to add 1.17.1 support

Gradle seems to have some problems with dependency resolution in this fork
